README.md

```diff
@@ -69,7 +69,10 @@ $ git clone https://github.com/google/googletest.git benchmark/googletest
 $ mkdir build && cd build
 # Generate a Makefile with cmake.
 # Use cmake -G <generator> to generate a different file type.
-$ cmake ../benchmark
```

Comment:
```suggestion
$ cmake ../
```

Reply:
I assumed one needs to cmake gtest first, but this is already enough. cmake finds googletest sources and does its thing.